APMrover2/mode_manual.cpp

```diff
@@ -1,12 +1,20 @@
 #include "mode.h"
 #include "Rover.h"
 
+void ModeManual::_exit()
+{
+    // clear lateral when exiting manual mode
+    g2.motors.set_lateral(0);
```

Comment:
The fact that this lateral control is only available in manual hints to me that we should reconsider omni Rovers as being Rovers.

I've previously mentioned this to Randy, but Omni rovers and boats with thrusters are more like quadcopters that are stuck to the ground than Planes stuck to the ground (which is kinda what our current Rovers are).

Reply:
Omni rovers and boats are similar to quadcopters and submarines but the limitation that they only move in 2D is a large one especially once we get to navigation.  I think it's too soon to try and merge them with multicopters.

Reply:
.. ah, maybe you weren't suggesting merging them.  In any case, I definitely agree that there's an opportunity for re-use of navigation controls between vehicles (especially navigation routines from copter moving into Rover).